README.md

```diff
@@ -0,0 +1,48 @@
+# Icinga DB
+
+![Icinga Logo](https://icinga.com/wp-content/uploads/2014/06/icinga_logo.png)
+
+#### Table of Contents
+
+- [About](#about)
+- [License](#license)
+- [Installation](#installation)
+- [Documentation](#documentation)
+- [Support](#support)
+- [Contributing](#contributing)
+
+## About
+
+Icinga DB serves as a synchronisation daemon between Icinga 2 (Redis) and Icinga Web 2 (MySQL). It synchronises configuration, state and history of an Icinga 2 environment using checksums.
+
+Icinga DB also supports reading from multiple environments and writing into a single MySQL instance.
+
+## License
+
+Icinga DB and the Icinga DB documentation are licensed under the terms of the GNU
+General Public License Version 2, you will find a copy of this license in the
+LICENSE file included in the source package.
```

Comment:
AFAIK we have GPLv2+. I'd Just use this one from the GPLv2:

```
This program is free software; you can redistribute it and/or
modify it under the terms of the GNU General Public License
as published by the Free Software Foundation; either version 2
of the License, or (at your option) any later version.
```

Reply:
I only copied the test from the Icinga 2 README. The key factor is the LICENSE file anyways.

Reply:
Then it's OK.